database/migrations/003.do.sql

```diff
@@ -0,0 +1,6 @@
+
+ALTER TABLE users
+  ALTER COLUMN name TYPE VARCHAR(255);
+
+ALTER TABLE organization_policies
```

Comment:
I saw you have tests for user's name length. Why don't you add a new one for `org_id`?